docs/api/search.md

```diff
@@ -93,22 +100,26 @@ This is an array containing exactly two values, which can be null or an integer
 
 Further information and understanding can be found by [reading the tests here](https://github.com/climatepolicyradar/navigator-backend/blob/ddebbd17f6b62cf7909e6e4c575285b8b00a41b2/tests/routes/test_search.py#L634).
 
-#### sort_field (optional) & sort_order (optional, defaults to descending)
+##### sort_field (optional) & sort_order (optional, defaults to descending)
 
 The field to sort by can be chosen from “date” or “title” [see related code](https://github.com/climatepolicyradar/navigator-backend/blob/ddebbd17f6b62cf7909e6e4c575285b8b00a41b2/app/api/api_v1/schemas/search.py#L20).
 
-The order can be chosen from ascending (use “asc”) or descending (use “desc”), [see related code](https://github.com/climatepolicyradar/navigator-backend/blob/ddebbd17f6b62cf7909e6e4c575285b8b00a41b2/app/api/api_v1/schemas/search.py#L13).
+The order can be chosen from ascending (use “asc”) or descending (use “desc”),
+[see related code.](https://github.com/climatepolicyradar/navigator-backend/blob/ddebbd17f6b62cf7909e6e4c575285b8b00a41b2/app/api/api_v1/schemas/search.py#L13)
 
-#### limit & offset
+##### limit & offset
 
-These values control pagination, allowing a front end application to page through the results. The `limit` refers to the maximum number of results to return and `offset` where to start returning the results from that were retrieved via the backend.
+These values control pagination, allowing a front end application to page
+through the results. The `limit` refers to the maximum number of results to
+return and `offset` where to start returning the results from that were
+retrieved via the backend.
 
+### **Response Payload**
 
-## **Response Payload**
+The response returns a list of families and includes their associated documents
+along with their passage matches. The payload has the following scheme:
 
-The response returns a list of families and includes their associated documents along with their passage matches. The payload has the following scheme:
-
-```
+```json
 {
   "hits": 0,
```

Comment:
This appears to show up in red as does the json below?: 

![image](https://github.com/climatepolicyradar/navigator-backend/assets/58440325/568b12e3-1dbb-44b9-825c-7cf8899c5630)

Reply:
Hmmm, I can't see that 🤔 
![Uploading image.png…]()

Reply:
![image](https://github.com/climatepolicyradar/navigator-backend/assets/46493669/ebf9f077-65d2-46bd-a4c1-310c76a0f1da)